### ISSUE TITLE
fix: Database is not stored in the right directory.

### ISSUE THREAD

**Issue by JcbSm** [JcbSm/Bridge#22]
On linux systems, the `.db` file will be created in the `/plugins/` root folder, instead of `/plugins/Bridge/`.

- Caused by directory separators being different

### Currently:
```
└── plugins/
    ├── Bridge/
    │   └── config.yml
    ├── Bridge\database.db
    └── Bridge-1.0.jar
```

### Ideally:
```
└── plugins/
    ├── Bridge/
    │   ├── config.yml
    │   └── database.db
    └── Bridge-1.0.jar
```

**Comment by JcbSm** [comment]
May require use of `File.separator`

**Comment by JcbSm** [comment]
Fixed in #31
